certbot/docs/using.rst

```diff
@@ -695,10 +695,47 @@ is done by means of a scheduled task which runs ``certbot renew`` periodically.
 
 If you are unsure whether you need to configure automated renewal:
 
-1. Review the instructions for your system at https://certbot.eff.org/instructions.
-   They will describe how to set up a scheduled task, if necessary.
-2. (Linux/BSD): Check your system's crontab (typically `/etc/crontab` and
-   `/etc/cron.*/*`) and systemd timers (``systemctl list-timers``).
+1. Review the instructions for your system and installation method at
+   https://certbot.eff.org/instructions. They will describe how to set up a scheduled task,
+   if necessary. If no step is listed, your system comes with automated renewal pre-installed,
+   and you should not need to take any additional actions.
+2. On Linux and BSD, you can check to see if your installation method has pre-installed a timer
+   for you. To do so, look for the ``certbot renew`` command in either your system's crontab
+   (typically `/etc/crontab` or `/etc/cron.*/*`) or systemd timers (``systemctl list-timers``).
+3. If you're still not sure, you can configure automated renewal manually by following the steps
+   in the next section. Certbot has been carefully engineered to handle the case where both manual
+   automated renewal and pre-installed automated renewal are set up.
+
+Setting up automated renewal
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+If you think you may need to set up automated renewal, follow these instructions to set up a
+scheduled task to automatically renew your certificates in the background. On the off chance you
+ended up at these instructions but automated renewal was pre-installed on your system, following
+these instructions should not cause any problems.
```

Comment:
I feel like we should avoid making any hedging or weaselly statements here. The instructions will just work!

My very quick attempt, but something else would probably be better:

```suggestion
scheduled task to automatically renew your certificates in the background. If you are unsure 
whether your system has a pre-installed scheduled task for Certbot, it is safe to follow these
instructions to create one.
```

Reply:
Wow, what a role reversal of our usual stances. But you're totally right, this is much better.